sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -530,7 +530,7 @@ The Analyze functionality allows to choose which of the supported Text Analytics
         DisplayName = "AnalyzeOperationSample"
     };
 
-    AnalyzeBatchActionsOperation operation = await client.StartAnalyzeBatchActionsAsync(batchDocuments, batchActions);
+    AnalyzeBatchActionsOperation operation = await client.StartAnalyzeBatchActionsAsync(batchDocuments, actions);
 
     await operation.WaitForCompletionAsync();
 
```

Comment:
Should we add operation metadata like expired on, created on etc?

Reply:
Added